src/Invoker.ts

```diff
@@ -46,13 +46,20 @@ export class Invoker {
                 send: (payload: any) => {
                     resolve(payload);
                     return response;
-                }
+                },
+                setHeader: () => {},
             };
 
             try {
-                const googleFunctionResponse = await Promise.resolve(googleFunction(jsonRequest, response));
+                const request = {
+                    body: jsonRequest,
+                    get: () => {},
+                    headers: {},
+                };
+                const googleFunctionResponse = await Promise.resolve(googleFunction(request, response));
+
                 if (googleFunctionResponse) {
-                   response.send(googleFunctionResponse);
+                    response.send(googleFunctionResponse);
```

Comment:
off indentation

Reply:
I'm correcting the indentation, we use 4 spaces, and that one had 3